Src/VimCore/StatusUtil.fs

```diff
@@ -16,16 +16,21 @@ type internal StatusUtil() =
         with get () = _vimBuffer
         and set value = _vimBuffer <- value
 
-    member x.DoWithBuffer func = 
+    member x.DoWithBuffer func (msg: string) = 
+        VimTrace.TraceInfo("Status Start\n{0}", msg)

```

Comment:
Should use `Environment.NewLine` or windows style new lines here.

Reply:
Right! Fixed.